tools/specs/lib/extractI3dmSpec.js

```diff
@@ -5,7 +5,7 @@ var extractI3dm = require('../../lib/extractI3dm');
 
 var fsReadFile = Promise.promisify(fs.readFile);
 
-var i3dmPath = './specs/data/instancedTile.i3dm';
+var i3dmPath = './specs/data/instancedWithTransform.i3dm';
```

Comment:
Instead use the i3dm in `Specs/Data/Cesium3DTiles/Instanced/InstancedWithBatchTable`, that way it won't be confusing if someone tries to load this i3dm without specifying a transform. You can still use the name `instancedTile.i3dm` though.